Gibe.Umbraco.Blog/BlogSections.cs

```diff
@@ -23,14 +22,16 @@ public BlogSections(IUmbracoWrapper umbracoWrapper, ISearchIndex searchIndex)
 		public IEnumerable<T> All()
 		{
 			var results = SearchForBlogSections();
-			return results.Select(r => Activator.Activate<T>(_umbracoWrapper.TypedContent(r.Id)));
+			return Enumerable.Empty<T>();
+			//return results.Select(r => Activator.Activate<T>(_umbracoWrapper.TypedContent(r.Id)));
```

Comment:
TODO?

Reply:
Depends on if we do the blog section stuff now or not.